tests/components/SingleAddon.test.js

```diff
@@ -95,40 +95,28 @@ describe('<SingleAddon/>', () => {
     expect(addonComponent.find("tr")).to.have.length(1);
   });
 
-  it('Should render 5 td tags', () => {
-    expect(addonComponent.find("td")).to.have.length(5);
+  it('Should render 4 td tags', () => {
+    expect(addonComponent.find("td")).to.have.length(4);
   });
 
   it('Should render 3 div tags', () => {
     expect(addonComponent.find("div")).to.have.length(3);
   });
 
-  it('Should render an image tag', () => {
-    expect(addonComponent.find("img")).to.have.length(1);
```

Comment:
Did this change require removing these tests?

Reply:
Yes it does. I got rid of the first column on the addon manager. That is the logo column.